operator/roles/forkliftcontroller/templates/ui-plugin/configmap-ui-plugin.yml.j2

```
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ app_name }}
+    service: {{ ui_plugin_service_name }}
+  name: nginx-conf
+  namespace: "{{ app_namespace }}"
+data:
+  nginx.conf: "error_log /dev/stdout info;\nevents {}\nhttp {\n\taccess_log         /dev/stdout;\n\tinclude            /etc/nginx/mime.types;\n\tdefault_type       application/octet-stream;\n\tkeepalive_timeout  65;\n\t\tserver {\n\t\t\tlisten              9443 ssl;\n\t\t\tssl_certificate     /var/serving-cert/tls.crt;\n\t\t\tssl_certificate_key /var/serving-cert/tls.key;\n\t\t\troot                /usr/share/nginx/html;\n\t\t}\n\t}\n"
```

can we use a `|` for the multi-line data ? 
```suggestion
  nginx.conf: |
    error_log /dev/stdout info;
    events {}
    ...
```